xbmc/windowing/CMakeLists.txt

```diff
@@ -9,7 +8,7 @@ set(HEADERS OSScreenSaver.h
             XBMC_events.h
             VideoSync.h)
 
-if(CORE_SYSTEM_NAME STREQUAL linux AND NOT MIR_FOUND)
+if(CORE_SYSTEM_NAME_LC STREQUAL rbpi OR CORE_SYSTEM_NAME_LC STREQUAL gbm)
```

Comment:
I think you want `CORE_PLATFORM_NAME` here

Reply:
more precisely `CORE_PLATFORM_NAME_LC`